app/src/main/java/com/duckduckgo/app/global/Theming.kt

```diff
@@ -34,13 +34,9 @@ enum class DuckDuckGoTheme {
 
 object Theming {
 
-    fun initializeTheme(settingsDataStore: SettingsDataStore, variantManager: VariantManager) {
+    fun initializeTheme(settingsDataStore: SettingsDataStore) {
         if (settingsDataStore.theme == null) {
-            if (variantManager.getVariant().hasFeature(VariantManager.VariantFeature.ConceptTest)) {
-                settingsDataStore.theme = DuckDuckGoTheme.LIGHT
-            } else {
-                settingsDataStore.theme = DuckDuckGoTheme.DARK
-            }
+            settingsDataStore.theme = DuckDuckGoTheme.LIGHT
```

Comment:
@CDRussell I'm removing this logic as you spotted that this could affect how we allocate users in some scenarios. From product, it's ok to go Light by default.